docs/program/get-started-SDKS.md

```diff
@@ -7,15 +7,16 @@ description: "Use Viam's SDKs to write code to access and control your robot."
 icon: "/services/img/icons/sdk.svg"
 tags: ["client", "sdk"]
 aliases:
-  - "product-overviews/sdk-as-client"
+  - "product-overviews/get-started-sdks"
+  - "program/get-started-sdks"
```

Comment:
the aliases are here so that old links that someone may have used still work:
```suggestion
  - "product-overviews/sdk-as-client"
  - "program/sdk-as-client"
```

Reply:
done